lorien/Main.gd

```diff
@@ -541,3 +541,4 @@ func _on_scale_changed() -> void:
 	# Canvas has to be set first to make it the correct scale on startup
 	_canvas.set_canvas_scale(scale)
 	get_tree().set_screen_stretch(SceneTree.STRETCH_MODE_DISABLED, SceneTree.STRETCH_ASPECT_IGNORE, Vector2(0,0), scale)
+	OS.min_window_size = Vector2(320 * scale, 256 * scale)
```

Comment:
Put the base min size in Config.gd like this: `const MIN_WINDOW_SIZE := Vector2(320, 256)`. Then multiply here with the scale.